tests/base/functions_ConstantVectorFunction_00.cc

```diff
@@ -0,0 +1,198 @@
+        logfile << std::endl;
+      }
+  }
+
+  logfile.close();
+  points.clear();
+
+  return;
+}
+
+// Construct ConstantVectorFunction with different constructor and test them
+template <int dim, typename Number>
+void test_constructor(const unsigned int n_component,
+                      const std_cxx11::array<Number, MAX_N_COMPONENT> &component_data)
+{
+  const unsigned int n_points(5);
+  // Instantiate with std::vector<>
+  {
+    std::ofstream logfile(LOG_FILE,std::fstream::app);
+    std::vector<Number> v(n_component);
+    for (unsigned int i=0; i<n_component; ++i)
+      v[i] = component_data[i];
+
+    logfile.setf(std::ios::fixed);
+    logfile << "\nIn " << dim << " dimension, "
+            << " Function with " << n_component << " components, "
+            << " Constructed with std::vector:" << std::endl;
+    // Close file to let Test use it.
+    logfile.close();
+    ConstantVectorFunction<dim> constant_vector_function(v);
+    test_one_object<dim, Number> (constant_vector_function, n_points, n_component);
+  }
+
+  // Instantiate with Vector
+  {
+    std::ofstream logfile(LOG_FILE,std::fstream::app);
+    Vector<Number> v(n_component);
+    for (unsigned int i=0; i<n_component; ++i)
+      v[i] = component_data[i];
+
+    logfile.setf(std::ios::fixed);
+    logfile << "\nIn " << dim << " dimension, "
+            << " Function with " << n_component << " components, "
+            << " Constructed with Vector:" << std::endl;
+    // Close file to let Test use it.
+    logfile.close();
+    ConstantVectorFunction<dim> constant_vector_function(v);
+    test_one_object<dim, Number> (constant_vector_function, n_points, n_component);
+  }
+
+  // Instantiate with pointer and offset
+  {
+    std::ofstream logfile(LOG_FILE,std::fstream::app);
+    logfile.setf(std::ios::fixed);
+    logfile << "\nIn " << dim << " dimension, "
+            << " Function with " << n_component << " components, "
+            << " Constructed with pointer and offset:" << std::endl;
+    // Close file to let Test use it.
+    logfile.close();
+    ConstantVectorFunction<dim> constant_vector_function(&component_data[0], n_component);
+    test_one_object<dim, Number> (constant_vector_function, n_points, n_component);
+  }
+
+  return;
+}
+
+// test different number of vector components
+template<int dim, typename Number>
+void test_n_components(const std_cxx11::array<Number, MAX_N_COMPONENT> &component_data)
+{
+#define N_COMPONENT_CASE 3
+  const std_cxx11::array<unsigned int, N_COMPONENT_CASE> components = { { 1, 4, 7 } };
+
+  for (unsigned int c=0; c<N_COMPONENT_CASE; ++c)
+    {
+      test_constructor<dim, Number> (components[c], component_data);
+    }
+
+  return;
+}
+
+int main()
+{
+  // Create data for each component
+  std_cxx11::array<NUMBER, MAX_N_COMPONENT> component_data =
+  { { 1, 255, 32768, 98.2, 75, 7.9e+8, 6.7e-4 } };
+
+  // Refresh file
+  // Not use deallog because Vector<Number>.print to logstream is deprecated.
```

Comment:
```deallog << v``` works.

But I didn't find the LogStream saying it has overloaded ```<<``` operator for Vector in its documents.